app/services/LoginService.php

```diff
@@ -44,4 +44,12 @@ function tryLogin(string $login, string $password, Validator &$v)
         logger()->info("Login failed: not found", ["login" => $login]);
         $v->set_error("Utilisateur non trouvé");
     }
+
+    function createActivationLink(User $user): string
```

Comment:
can you rename this file to `AuthService` (to make it consistent)

Reply:
✅